diff --git a/debian/indi-ocs/changelog b/debian/indi-ocs/changelog
index 5a86e3a..78f7716 100644
--- a/debian/indi-ocs/changelog
+++ b/debian/indi-ocs/changelog
@@ -2,13 +2,13 @@ indi-ocs (1.3) nobel; urgency=medium
 
   * New release - fix for park state and typo
 
--- Ed Lee <ed@thefamilee.co.uk> Sun, 08 Jun 2025 20:00:00 +0100
+ -- Ed Lee <ed@thefamilee.co.uk>  Sun, 08 Jun 2025 20:00:00 +0100
 
 indi-ocs (1.2) nobel; urgency=medium
 
   * New release - adds support for shutter control on park/unpark & dome Az slaving
 
--- Ed Lee <ed@thefamilee.co.uk>  Mon, 26 May 2025 15:15:00 +0100
+ -- Ed Lee <ed@thefamilee.co.uk>  Mon, 26 May 2025 15:15:00 +0100
 
 indi-ocs (1.1) nobel; urgency=medium
 
