diff --git a/debian/indi-eqmod/control b/debian/indi-eqmod/control
index 281e259..e95e0c0 100644
--- a/debian/indi-eqmod/control
+++ b/debian/indi-eqmod/control
@@ -2,7 +2,7 @@ Source: indi-eqmod
 Section: science
 Priority: extra
 Maintainer: Jasem Mutlaq <mutlaqja@ikarustech.com>
-Build-Depends: debhelper (>= 6), cdbs, cmake, libindi-dev, zlib1g-dev, libnova-dev, libgsl-dev, libahp-gt
+Build-Depends: debhelper (>= 6), cdbs, cmake, libindi-dev, zlib1g-dev, libnova-dev, libgsl-dev
 Standards-Version: 3.9.2
 
 Package: indi-eqmod
