diff --git a/debian/libplayerone/changelog b/debian/libplayerone/changelog
index 0ce58ae..1350f34 100644
--- a/debian/libplayerone/changelog
+++ b/debian/libplayerone/changelog
@@ -2,6 +2,8 @@ libplayerone (1.16) bionic; urgency=low
 
   * Update PlayerOneCamera SDK to v3.9.0
 
+ -- Hiroshi SAITO <hiro3110g@gmail.com>  Sat, 5 Apr 2025 08:00:00 +0900
+
 libplayerone (1.15) bionic; urgency=low
 
   * Update PlayerOneCamera SDK to v3.8.1
