diff --git a/debian/indi-playerone/changelog b/debian/indi-playerone/changelog
index 9167d6e..08d50fb 100644
--- a/debian/indi-playerone/changelog
+++ b/debian/indi-playerone/changelog
@@ -2,6 +2,8 @@ indi-playerone (1.20) bionic; urgency=low
 
   * Update PlayerOneCamera SDK v3.9.0
 
+ -- Hiroshi SAITO <hiro3110g@gmail.com>  Sat, 5 Apr 2025 12:00:00 +0900
+
 indi-playerone (1.19) bionic; urgency=low
 
   * Update PlayerOneCamera SDK v3.8.1
