diff --git a/debian/indi-playerone/changelog b/debian/indi-playerone/changelog
index d756fcf..29af972 100644
--- a/debian/indi-playerone/changelog
+++ b/debian/indi-playerone/changelog
@@ -2,10 +2,14 @@ indi-playerone (1.21) bionic; urgency=low
 
   * Update PlayerOneCamera SDK v3.9.1
 
+ -- Hiroshi SAITO <hiro3110g@gmail.com>  Sat, 15 Nov 2025 12:00:00 +0900
+
 indi-playerone (1.20) bionic; urgency=low
 
   * Update PlayerOneCamera SDK v3.9.0
 
+ -- Hiroshi SAITO <hiro3110g@gmail.com>  Fri, 5 Sep 2025 12:00:00 +0900
+
 indi-playerone (1.19) bionic; urgency=low
 
   * Update PlayerOneCamera SDK v3.8.1
