diff --git a/debian/indi-limesdr/control b/debian/indi-limesdr/control
index 6a48f2c..629eecf 100644
--- a/debian/indi-limesdr/control
+++ b/debian/indi-limesdr/control
@@ -2,7 +2,7 @@ Source: indi-limesdr
 Section: science
 Priority: extra
 Maintainer: Jasem Mutlaq <mutlaqja@ikarustech.com>
-Build-Depends: debhelper (>= 6), cdbs, cmake, libindi-dev, zlib1g-dev, libusb-1.0-0-dev, limesuite,  libcfitsio3-dev|libcfitsio-dev
+Build-Depends: debhelper (>= 6), cdbs, cmake, libindi-dev, zlib1g-dev, libusb-1.0-0-dev, liblimesuite-dev,  libcfitsio3-dev|libcfitsio-dev
 Standards-Version: 3.9.2
 
 Package: indi-limesdr
