diff --git a/debian/indi-ocs/changelog b/debian/indi-ocs/changelog
index 5a86e3a..78f7716 100644
--- a/debian/indi-ocs/changelog
+++ b/debian/indi-ocs/changelog
@@ -2,13 +2,13 @@ indi-ocs (1.3) nobel; urgency=medium
 
   * New release - fix for park state and typo
 
--- Ed Lee <ed@thefamilee.co.uk> Sun, 08 Jun 2025 20:00:00 +0100
+ -- Ed Lee <ed@thefamilee.co.uk>  Sun, 08 Jun 2025 20:00:00 +0100
 
 indi-ocs (1.2) nobel; urgency=medium
 
   * New release - adds support for shutter control on park/unpark & dome Az slaving
 
--- Ed Lee <ed@thefamilee.co.uk>  Mon, 26 May 2025 15:15:00 +0100
+ -- Ed Lee <ed@thefamilee.co.uk>  Mon, 26 May 2025 15:15:00 +0100
 
 indi-ocs (1.1) nobel; urgency=medium
 
diff --git a/debian/libfli/control b/debian/libfli/control
index 1beee1e..3789a74 100644
--- a/debian/libfli/control
+++ b/debian/libfli/control
@@ -30,6 +30,4 @@ Priority: extra
 Section: debug
 Architecture: any
 Depends: libfli2 (= ${binary:Version}), ${misc:Depends}
-Description: Finger Lakes Instruments (FLI) debug symbols.
-.
-This package contains debug symbols.
+Description: Finger Lakes Instruments (FLI) debug symbols. This package contains debug symbols.
diff --git a/debian/libpigpiod/libpigpiod.install b/debian/libpigpiod/libpigpiod.install
index 9c8591a..e34e861 100644
--- a/debian/libpigpiod/libpigpiod.install
+++ b/debian/libpigpiod/libpigpiod.install
@@ -1,3 +1,3 @@
 usr/bin
 usr/include
-usr/lib/systemd/system
+lib/systemd/system
